packages/core/src/application.ts

```diff
@@ -50,6 +52,8 @@ export class Application extends Context {
       this._handleHttpRequest(req, res);
 
     this.bind('logError').to(this._logError.bind(this));
+    this.bind('send').to(writeResultToResponse);
```

Comment:
I start to wonder if we should use a prefix for such actions, for example:
`actions.send` or even `sequence.actions.send`.